[CAS-948] Global Customization Documentation (#1835)
GetStream/stream-chat-android: documentation/docs/ui/global_customizations/chat_ui_customizations/markdown.md

---
id: uiGlobalCustomizationsMarkdown
title: Markdown
sidebar_position: 5
---

> The android SDK already was Markdown support by default, by you add your own Markdown parser. Implement the `ChatMarkdown` interface:

Reviewer: ```suggestion
The Android SDK already has Markdown support by default. You can modify it by implementing a custom `ChatMarkdown` interface:
```
